src/main/resources/styles/reporting.css

```diff
@@ -169,9 +166,10 @@ table.stats-table tr.header {
 
 table.stats-table tr.summary {
 	background-color: lightgray;
+	font-weight: bold;
 }
 
-table.stats-table td.duration {
+table.stats-table td.duration-format {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (td.duration-format) is overqualified, just use .duration-format without element name.](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2040468999/issues/source?bid=3059308&fileBranchId=3130319#l172)